lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -27,6 +27,7 @@ $(document).ready(function() {
     const slides = [InstructionSlide]; // ReportbackItemKudosSlide, ContextSlide
 
     $('.chrome').find('> .wrapper').before(`<div id="${jsxElementHook}"></div>`);
+    $('.messages').remove();
```

Comment:
had to remove the - from the error & add missing quote but otherwise it worked!

<img width="389" alt="screen shot 2016-07-27 at 12 30 25 pm" src="https://cloud.githubusercontent.com/assets/897368/17186081/bb728c44-5401-11e6-99d7-c98a1ecacf12.png">